pom.xml

```diff
@@ -262,6 +262,16 @@
             <artifactId>jsoup</artifactId>
             <version>1.10.1</version>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.owasp-java-html-sanitizer</groupId>
+            <artifactId>owasp-java-html-sanitizer</artifactId>
+            <version>20160924.1</version>
+        </dependency>
+        <dependency>
```

Comment:
Owasp depends on guava. It's listed in the parent pom of owasp as 
```
      <dependency>
        <groupId>com.google.guava</groupId>
        <artifactId>guava</artifactId>
        <!--
            We depend on 11 because it is the last which works with JDK 5
            which we want to link against.
        -->
        <version>[11.0,)</version>
        <scope>provided</scope>
      </dependency>
```